FWCore/Services/web/README.md

```diff
@@ -22,7 +22,7 @@ open -a Google\ Chrome --args --allow-file-access-from-files file://<where ever
 Local file support in Safari requires access the the Developer menu options. You can turn on those options by going to `Safari/Settings...`, choose the `Advanced` tab and then toggle `Show features for web developers`. Then in the newly added `Developer` tag, toggle `Disable local file restrictions`. Once you've done that, you can use `file://<where ever the directory is>/index.html` to start the application.
 
 ## Viewer layout
-The viewer is composed to two main areas, the top is the timing viewer and at the bottom shows information about a selected time block. The top time viewer is further divided into three parts. On the left is the macro scoping grouping of framework activity types into Global and Stream activities. At the bottom is the measurement of time since the start of the job. The main area shows the blocks of time spend doing various work within the Framework.
+The viewer is composed to two main areas, the top is the timing viewer and at the bottom shows information about a selected time block. The top time viewer is further divided into three parts. On the left is the macro scoping grouping of framework activity types into Global and Stream activities. If the `module centric` option was chosen then this area is also divided by each module activity which is sorted based on most time used to least time used. At the bottom is the measurement of time since the start of the job. The main area shows the blocks of time spend doing various work within the Framework.
```

Comment:
Would `"module centric"` or `_module centric_` provide better formatting? (I was just surprised to see the code formatting with backquotes for it)